WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -626,4 +643,18 @@ private extension ShippingLabelFormViewModel {
         static let orderSummaryHeader = NSLocalizedString("Shipping label order summary",
                                                           comment: "Header of the order summary section in the shipping label creation form")
     }
+
+    enum Constants {
+        static let acceptedUSPSCountries = [
```

Comment:
Probably good to add [the link](https://github.com/Automattic/woocommerce-services/blob/dbeed610f336f8db7995743765dbcd0ff2e18474/client/extensions/woocommerce/woocommerce-services/state/shipping-label/constants.js#L1) as a comment so it's easier to keep them in sync!